packages/admin/cms-admin/src/redirects/RedirectForm.tsx

```diff
@@ -112,13 +113,14 @@ export const RedirectForm = ({ mode, id, linkBlock, scope }: Props): JSX.Element
 
             const { data } = await client.query<GQLRedirectSourceAvailableQuery, GQLRedirectSourceAvailableQueryVariables>({
                 query: gql`
-                    query RedirectSourceAvailable($scope: RedirectScopeInput!, $source: String!) {
-                        redirectSourceAvailable(scope: $scope, source: $source)
+                    query RedirectSourceAvailable($scope: RedirectScopeInput!, $source: String!, $excludedId: ID) {
+                        redirectSourceAvailable(scope: $scope, source: $source, excludedId: $excludedId)
```

Comment:
In the page tree we check this in the Admin, should we do this here as well?
https://github.com/vivid-planet/comet/blob/app-router-predefined-pages/packages/admin/cms-admin/src/pages/createEditPageNode.tsx#L149-L152

Reply:
I just looked at the problem again and realised that I completely misunderstood it 😅

The described behavior only occurred on the add page. The real reason was that there was no redirect from the add page to the edit page after creating the redirect. Therefore, the validation failed. My initial "fix" actually only hid this problem.

Now I correctly redirect to the edit page after creation